overlord/snapstate/backend_test.go

```diff
@@ -206,6 +206,14 @@ func (f *fakeSnappyBackend) RemoveSnapData(info *snap.Info) error {
 	return nil
 }
 
+func (f *fakeSnappyBackend) RemoveSnapSharedData(info *snap.Info) error {
+	f.ops = append(f.ops, fakeOp{
+		op:   "remove-snap-shared-data",
+		name: info.MountDir(),
```

Comment:
You mean because it includes the version? Perhaps this is reason enough to create an `info.BaseDir()` function, then.